security_monitoring/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# CHANGELOG - security_monitoring
+
```

Comment:
There should probably be a changelog, something like
```
## 1.0.0 / 2021-19-05
* Security monitoring saved view
```

Reply:
I think the changelog is filled in automatically from the PR title

Reply:
Yes changelog is updated automatically when we perform individual integration releases